addon/components/horizontal-list-swiper/gesture/component.js

```diff
@@ -6,22 +6,34 @@ import Ember from 'ember';
 import RecognizerMixin from 'ember-gestures/mixins/recognizers';
 import layout from './template';
 import { run } from '@ember/runloop';
+import {getOwner} from '@ember/application';
 
 const {
   String
 } = Ember;
 
 export default Component.extend(RecognizerMixin, {
-
+  window: window,
   layout,
   viewport: service(),
```

Comment:
I guess this service has also been moved to open source?

Reply:
I think we should create a separate open-source addon to share usual tree-shaked services. Added as a bullet point (this service shared in this repo for now).